Resources/Locale/_RMC14/reagents/meta/elements.ftl

```diff
@@ -0,0 +1,2 @@
+reagent-name-sulphuric-rmc-acid = sulphuric acid
+reagent-desc-sulphuric-rmc-acid = A corrosive chemical. Keep away from your face.
```

Comment:
```suggestion
reagent-name-rmc-sulphuric-acid = sulphuric acid
reagent-desc-rmc-sulphuric-acid = A corrosive chemical. Keep away from your face.
```

Reply:
WAIT, why is that file in `Resources/Locale/_RMC14/` ??? move it to `Resources/Locale/en-US/_RMC14`